src/t2iapi/metric/service.proto

```diff
@@ -35,10 +36,12 @@ service MetricService {
         returns (BasicResponse);
 
     /*
-    Change the value for the metric with the provided handle and confirm the value using the device ui.
-     */
-    rpc SetUserConfirmableValue (BasicHandleRequest)
-        returns (BasicResponse);
+    For the metric with the given handle do the following at once:
+     - set the @ActivationState to the given activation value
+     - change the MetricValue/@Value and confirm the value using the device ui
+   */
+   rpc SetActivationStateAndUserConfirmableValue (t2iapi.activation_state.SetComponentActivationRequest)
```

Comment:
make a new request in metric_requests.proto
this request should get a string handle and a ComponentActivation activation
import "t2iapi/activation_state/activation_state_requests.proto" in metric_requests.proto

Reply:
I run into some import issues, thats why i created a new enum of activationstates besides the new request.
This is kind of doubling but reduces the mixing of packages.

We may discuss this again.